util/cache/cache.go

```diff
@@ -0,0 +1,57 @@
+package cache
```

Comment:
Can we move this to github.com/evergreen-ci/utility? The `util` package was supposed to be largely replaced by utility.

Reply:
Sure